### DISCUSSION TITLE
I cannot a run a Nuclei for a Single domain, I'm getting an error

### DISCUSSION THREAD

**Discussion by sherli297** [projectdiscovery/nuclei-templates#9083] [Q&A] [answered]
I'm not able to run a single domain or multiple domains, I'm getting an error please find the error screenshot for reference. Please do help me to sort this issue. The Nuclei version is  v3.1.9
![nuclei error](https://github.com/projectdiscovery/nuclei-templates/assets/85586714/0cacde87-1366-4d6d-9c2d-d4920a55f33d)

**Comment by ehsandeep** [comment, marked as answer]
@sherli297 update to latest version - https://github.com/projectdiscovery/nuclei/releases/tag/v3.1.10
